Builds/levelization/levelization.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
```

Comment:
What is the interface of this script? Does it assume a particular working directory? Is it customizable through any environment variables or command line arguments or input? If so, can we document that interface at the top in a comment?

Reply:
Good question. It has no options, and can be run from anywhere, as long as the script is in it's correct place in the repo tree. That's what line 3: `pushd $( dirname $0 )` does.

Reply:
"Usage" added to the script.